File: Length of Last Word/Length of Last Word.java
public class Solution {
    public int lengthOfLastWord(String s) {
        int lastlen=0;
        int i=s.length()-1;
        while((i>=0)&&s.charAt(i)==' '){
        --i;
        }
        while((i>=0)&&s.charAt(i)!=' '){
            ++lastlen;
            --i;
        }  
        return lastlen;
    }
}

File: Length of Last Word/Length of Last Word.py
class Solution:
    # @param s, a string
    # @return an integer
    def lengthOfLastWord(self, s):
        last=0
        i=len(s)-1
        while i>=0 and s[i]==' ':
            i-=1
        while i>=0 and s[i]!=' ':
            i-=1
            last+=1
        return last
